### ISSUE TITLE
How to test?

### ISSUE THREAD

**Issue by Seeseallllll** [MCG-NJU/VideoMAE-Action-Detection#4]
Thanks for your work

After fine-tuning, the performance of the model on the verification set is obtained, but how to get the accuracy of the model on the test set? Is there a demo file or process for reference?

**Comment by Sleepingbug** [comment]
+1，文档写得太粗糙了。文档说了 该项目基于 MAE-pytorch、BEiT 和 AlphAction 构建。好歹也学学 MAE-pytorch，起码还有一句 python run_mae_vis.py ${IMAGE_PATH} ${OUTPUT_DIR} ${MODEL_PATH} 让人知道怎么测试，而这里没有。
